### PR TITLE
[Gles20] Silenced minor deprecation warning.

### PR DESCRIPTION
**Pull request by donaldalanmorrison**
UITextAlignment soon NSTextAlignment.

### CODE REVIEW

File: tools/cxx-generator [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit db61f8399655dd90f09d550259b45ae8289364fe
+Subproject commit 5154f8898bb8755bda496a84fa39abe97d6a9b44
```
Comment (minggo) [inline tools/cxx-generator:1]: Please don't change this submodule address.

Comment (donaldalanmorrison) [inline tools/cxx-generator:1]: 抱歉。 I didn't make the change, I pulled it from upstream.  Is there an alternative explanation?

**Comment by minggo** [comment]
Yep, it seems that this file will be modified every time.
I don't know the reason.

**Comment by donaldalanmorrison** [comment]
Hmmm...  In ".gitmodules":

[submodule "tools/cxx-generator"]
        path = tools/cxx-generator
        url = git://github.com/funkaster/cxx-generator.git

So someone did a "git submodule update".... I'm pretty sure it wasn't me.  Sorry!

**Comment by donaldalanmorrison** [comment]
I think dualface accidentally pulled it:

https://github.com/dualface/cocos2d-x/commits/gles20

https://github.com/dualface/cocos2d-x/commit/7a4cd77d512999af83f1a475339f50acece6ec51

https://github.com/dualface/cocos2d-x/commit/728a656a5d1bc444c8c5fb46c3c796bfbd3f4348

**Pull request merged**